Automattic/wp-calypso
# Jetpack pricing page: update footer

### Changes proposed in this Pull Request

This PR updates the footer of the Jetpack pricing page to match the new [jetpack.com](http://jetpack.com/) footer.

### Implementation notes

- Language toggle was not implemented in the previous footer. It'll be implemented here in a separate PR.
- The logic to hide the California notice for non Californian users will be implemented in a later PR too.

### Testing instructions

- Download this PR and run cloud
- Visit `/pricing/`
- Check that the footer is similar to the Jetpack.com footer, keeping in mind the two points above
- Check that each link directs you to the page it advertises
- Check that tracking events are triggered when clicking a link

### Screenshots
<img width="1332" alt="Screen Shot 2022-02-02 at 2 21 41 PM" src="https://user-images.githubusercontent.com/1620183/152222253-56b052de-e19b-4f8d-9b1e-e4688f26640b.png">
